internal/pkg/utils/utils.go

```diff
@@ -454,15 +454,10 @@ func GetIdentifiers(args *skel.CmdArgs, nodename string) (*WEPIdentifiers, error
 	return &epIDs, nil
 }
 
-func GetHandleID(netName string, containerID string, workload string) (string, error) {
+func GetHandleID(netName, containerID string) string {
 	handleID := fmt.Sprintf("%s.%s", netName, containerID)
-	logrus.WithFields(logrus.Fields{
```

Comment:
I think we do actually want to keep this log as it is - it's really useful to have all this information in the CNI log output.

Reply:
No problem reverting the change, but isn't it sufficient to just log out the handleID since it's constructed using the netName and containerID?

Reply:
I think the main thing is we want a log which includes the WorkloadID as well so that we can tie the two together.